lib/awful/layout/suit/tile.lua

```diff
@@ -49,7 +49,9 @@ tile.resize_jump_to_corner = true
 local function mouse_resize_handler(c, _, _, _, orientation)
     orientation = orientation or "tile"
     local wa = c.screen.workarea
-    local mwfact = c.screen.selected_tag.master_width_factor
+    local t = c.screen.selected_tag
+    local useless_gap = t.gap
+    local mwfact = t.master_width_factor
```

Comment:
I'm not sure whether this is correct. What happens when the user selects multiple tags?

As a possible fix, we could add a `params` parameter like the one for `arrange()`.

Reply:
that's existing code, see the next line of the diff - i've just assigned it to the variable to avoid writing c.screen..... twice

Reply:
answering your question - having two tags (with different params) selected in awesome i think not documented behavior, but i think everywhere it defaults to params of the first (by idx) of selected tags - although this PR is not the best place to discuss that behavior